package.json

```diff
@@ -7,7 +7,7 @@
     "iot",
     "device"
   ],
-  "version": "5.12.0",
+  "version": "5.12.1",
```

Comment:
How did this version change in this PR branch? This doesn't seems right.

Reply:
We should revert this. It will confuse semantic release. Versions are automatically released off of main when PRs are merged